oracle/oracle.js

```diff
@@ -1,58 +1,60 @@
 require('dotenv').config()
-const cosmosjs = require("@cosmostation/cosmosjs");
+const kava = require('@kava-labs/javascript-sdk');
 const CoinGecko = require('coingecko-api');
 const cron = require('node-cron');
 
-import { postTxKava, getTxKava } from '../common/txs.js';
-import { newMsgPostPrice } from '../common/msg.js';
-import { loadCoinNames } from './utils.js';
+const coinUtils = require('./utils.js');
 
-// Load chain details, credentials
-const mnemonic = process.env.MNEMONIC
-const lcdURL = process.env.LCD_URL
-const chainID = process.env.CHAIN_ID;
 
-// Load params
-const marketIDs = process.env.MARKET_IDS.split(",");
-const coinNames = loadCoinNames(marketIDs)
+var main = async () => {
+    // Load chain details, credentials
+    const mnemonic = process.env.MNEMONIC
+    const lcdURL = process.env.LCD_URL
+    const chainID = process.env.CHAIN_ID;
```

Comment:
`chainID` is not used and can be removed. It can likely be removed from .env as well, as the client loads chain ID by making a query to the rest server on `initChain()`

Reply:
We may want have it be an optional param to `initChain()`, otherwise user's could connect to a remote server that changed it's chain-id and not know.